lib/Wikia/src/Util/Consul/ExternalServicesQuery.php

```diff
@@ -0,0 +1,40 @@
+<?php
+namespace Wikia\Util\Consul;
+use Wikia\Logger\WikiaLogger;
+
+/**
+ * Class for querying Wikia Microservices provided by Consul
+ * TODO: do actual request functionality
```

Comment:
We probably need something that will handle the communication between MW and services (logging, retrying, fail handling). As this topic is much broader that our small project - I decided to not implement the functionality at the moment.